mass_mailing_custom_unsubscribe_event/demo/assets.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - João Marques
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="assets_tests"
+          inherit_id="web.assets_tests">
+    <xpath expr=".">
+        <script type="text/javascript"
+                src="/mass_mailing_custom_unsubscribe_event/static/src/js/tour.js"/>
+    </xpath>
+</template>
+
+<!--This is set here to make tours work-->
+<template id="mass_mailing_custom_unsubscribe.layout"
+          inherit_id="mass_mailing.layout">
+    <t t-call-assets="web.conditional_assets_tests"/>
```

Comment:
So, wouldn't this make sense in `mass_mailing_custom_unsubscribe` directly?

AFAICS it is still using the old v12 trick of loading test assets as demo assets too: https://github.com/OCA/social/blob/df91582a694e45d4499ed3fb5e02c5f606f04cff/mass_mailing_custom_unsubscribe/__manifest__.py#L21

If you fix it in that addon, then this one doesn't need to add this to the template and will work the same. Of course, line 8 should use `inherit_id="mass_mailing_custom_unsubscribe.assets_tests"` instead once done.